index.js

```diff
@@ -23,13 +23,34 @@ module.exports.pitch = function (request) {
 		"// Hot Module Replacement",
 		"if(module.hot) {",
 		"	// When the styles change, update the <style> tags",
-		"	if(!content.locals) {",
```

Comment:
When I removed this check, I dedented the code so the diff looks a little bit larger than it is.